package.json

```diff
@@ -68,8 +69,7 @@
     "redux": "^3.7.2",
     "rimraf": "^2.6.3",
     "sinon": "^2.4.1",
-    "typescript": "^3.2.2",
-    "typescript-eslint-parser": "^21.0.2"
+    "typescript": "^3.2.2"
```

Comment:
I would expect this version to be pinned. TypeScript don't follow SemVer.

Reply:
sure, fair enough:
```suggestion
    "typescript": "~3.2.2"
```